src/components/Navbar.tsx

```diff
@@ -1,24 +1,18 @@
-export const Navbar = () => {
-  return (
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">Home</a>
+import { FC } from 'react';
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
 
-          <a
-            aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
-  );
+type Props = {
+  page: string,
+  to: string,
 };
+
+export const Navbar: FC<Props> = ({ page, to }) => (
```

Comment:
The component name `Navbar` does not accurately represent its functionality. This component now represents a single link, not a navigation bar. Consider renaming it to something more accurate, like `NavbarLink`.